python/paddle/tensor/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .attribute import shape  # noqa: F401
 from .attribute import real  # noqa: F401
 from .attribute import imag  # noqa: F401
+from .attribute import is_floating_point  # noqa: F401
```

Comment:
I checked the code of `is_floating_point`, do we have to add API document for this?

Reply:
done